exercise_11.js

```diff
@@ -0,0 +1,11 @@
+/*
+Complete the square sum function so that it squares each number passed into it and then sums the results together.
+
+For example, for [1, 2, 2] it should return 9 because 1**2 + 2**2 + 2**2 = 9.
+ */
+
+function squareSum(numbers){
```

Comment:
Please don't do exercises that are out of reach for your current knowledge, I'm almost sure that you don't understand the code here, so there is no point for you to copy such a code from StackOverflow. Let me know if I'm wrong.